src/repositories/user.repository.ts

```diff
@@ -50,4 +50,26 @@ export class UserRepository {
     });
     return productList;
   }
+  public async getProductById(index: number): Promise<IProduct> {
+    const productFound = await this.db.query({
+      sql: `SELECT * FROM products WHERE productId= ${index}`,
+    });
+    return productFound[0];
+  }
+  public async saveProduct(product: IProduct, id: number): Promise<IProduct> {
+    const okPacket: OkPacket = await this.db.query({
+      sql: `INSERT INTO products (name, description, imagePath, price, id) VALUES('${product.name}', '${product.description}', '${product.imagePath}', '${product.price}', '${id}');`,
```

Comment:
SQL injection here as well, looks very bad to use the driver this way.

Reply:
This is the same exact way as we did it in the user feature. Can you please give me a hand?

Reply:
Its a temporary solution until the architecture of the front end is done. thnaks